<launch>
	<!-- Keyboard Teleop Node -->
	<node pkg="robotino_teleop" type="keyboard_teleop_node"
		name="keyboard_teleop_node" output="screen" >
				
		<!-- Scale -->
		<param name="scale_linear" value="10.0" type="double" />
		<param name="scale_angular" value="2.0" type="double" />
	</node>
</launch>